chef/cookbooks/barclamp/libraries/barclamp_library.rb

```diff
@@ -160,6 +163,15 @@ def self.claimed(node,owner)
           end
         end
 
+        def self.multipath?(device)
+          uuid_path = "/sys/block/#{device}/dm/uuid"
+          if File.exist?(uuid_path) && File.open(uuid_path) { |f| f.read(7).start_with?("mpath-") }
```

Comment:
Line is too long.  Also the `true` / `false` in the conditional body lends itself to a rewrite, e.g.

    return false unless File.exist?(uuid_path)
    File.open(uuid_path) { |f| f.read(7).start_with?("mpath-") }